### ISSUE TITLE
Scan item

### ISSUE THREAD

**Issue by ShootterDev** [kolton/d2bot-with-kolbot#2783]
how do i config, kolbot for only creat 1 game and scan item take wp come back town, check again till i have what i want

Drognan Normal game!

Do only that until i stop him

**Comment by DarkHorseDre** [comment]
you should read the character config in general as it explains the supported features.

in particular search for `Scripts.ShopBot` and read the comments about how to configure it

**Comment by ShootterDev** [comment]
yes but when i check this, its only when he go town or at the start... 
i want a script like he do only that during xx Hours ^^

**Comment by ShootterDev** [comment]
if u can help to find something :  to do White runeword DOL + IO
So i search base like : +3 +3 u know what i mean?
And its sell by Drognan Act2 Normal Game

**Comment by DarkHorseDre** [comment]
no, shopbot runs continuously if you configure it as written.

so make shopbot active, edit the nip it indicates, and specify drognan as the npc and you're done.

it's easy - Ensure you read the config lines for shopbot properly
